feat: Add a logger to embed init options (#732)
vega/vega-embed: README.md

| `defaultStyle` | Boolean or String | If set to `true` (default), the embed actions are shown in a menu. Set to `false` to use simple links. Provide a string to set the style sheet. |
| `bind`        | String or Element | The element that should contain any input elements bound to signals. |
| `renderer`    | String        | The renderer to use for the view. One of `"canvas"` (default) or `"svg"`. See [Vega docs](https://vega.github.io/vega/docs/api/view/#view_renderer) for details. May be a custom value if passing your own `viewClass` option. |

> | `logger`      | Object        | Sets a custom logger handler. See [Vega docs](https://vega.github.io/vega/docs/api/view/#view_logger) for details. It must follow the interface of [the original one](https://github.com/vega/vega/tree/master/packages/vega-util#logger). |

Reviewer: ```suggestion
| `logger`      | Object        | Sets a [custom logger handler on the view](https://vega.github.io/vega/docs/api/view/#view_logger). A logger must implement the [Vega logger API](https://github.com/vega/vega/tree/master/packages/vega-util#logger). |
```
